### PR TITLE
docs: tooltip [OR-589]

### PR DESCRIPTION
**Pull request by itsdeyan**
just an experiment

## Checklist before requesting a review
- [ ] I have applied `percy` label on my PR before merging and after review.
- [ ] If it is a new feature, I have added thorough tests.
- [ ] I have updated relevant docs.
- [ ] I have updated relevant env variables in Doppler.

### CODE REVIEW

File: apps/for-everyone-website/src/content/docs/components/tooltip-onboarding.mdx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+- Contain only descriptive unformatted text with an optional title for emphasis.
+- Should provide insights or tips that enhance, but aren't critical to, user understanding.
+- Appear only once; dismissing them removes them permanently.
+
+## Anatomy
+
+<OnboardingTooltipAnatomy />
+
+1. <strong>Heading (optional)</strong>: Generally, it's best to omit the heading to maintain simplicity and accessibility. Employ a heading only when it's imperative for understanding.
+2. <strong>Text</strong>: The main content pro  viding detailed explanation or information.
```
Comment (frshwtr) [inline apps/for-everyone-website/src/content/docs/components/tooltip-onboarding.mdx:22]:
```suggestion
2. <strong>Text</strong>: The main content providing detailed explanation or information.
```

File: apps/for-everyone-website/src/content/docs/components/tooltip-onboarding.mdx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+- Contain only descriptive unformatted text with an optional title for emphasis.
+- Should provide insights or tips that enhance, but aren't critical to, user understanding.
+- Appear only once; dismissing them removes them permanently.
+
+## Anatomy
+
+<OnboardingTooltipAnatomy />
+
+1. <strong>Heading (optional)</strong>: Generally, it's best to omit the heading to maintain simplicity and accessibility. Employ a heading only when it's imperative for understanding.
```
Comment (notlee) [inline apps/for-everyone-website/src/content/docs/components/tooltip-onboarding.mdx:21]: We can't emphasis the heading to assistive tech, which means we should never use a heading if it is imperative for understanding. How about?

```suggestion
1. <strong>Heading (optional)</strong>: It's best to omit the tooltip heading as emphasis cannot be portrayed to screenreader users. Employ a heading only for presentational reasons, provided emphasis is not required for understanding.
```

**Comment by notlee** [comment]
Spoke with Deyan about the above. He's happy with those suggestions which I offered to make :)

**Review by notlee — APPROVED** [review]

**Comment by notlee** [comment]
Closing in favour of https://github.com/Financial-Times/origami/pull/1481

**Pull request closed without merging**